src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -53,6 +62,8 @@ export class BuildGraphTableComponent implements OnChanges {
   public locked: boolean = false;
   public focusedBuildId?: number;
 
+  getRepo = getRepo;
```

Comment:
Is this correct?

Reply:
yes, it assigns the function to the component class so the view can access it.